src/Fabulous/IViewElement.fs

```diff
@@ -0,0 +1,23 @@
+// Copyright Fabulous contributors. See LICENSE.md for license.
+namespace Fabulous
+
+open System
+open Fabulous.Tracing
+
+type ProgramDefinition =
+    { CanReuseView: IViewElement -> IViewElement -> bool
+      Dispatch: obj -> unit
+      TraceLevel: TraceLevel
+      Trace: TraceLevel * string -> unit }
+
+and IViewElement =
+    abstract Create: ProgramDefinition -> obj
```

Comment:
I guess `///` comments on each of these would be good :-) Given the central nature of the interface :)

Reply:
Yes, I'm planning to put `///` comments on all public surface APIs.